src/main/java/teammates/common/util/Const.java

```diff
@@ -1148,7 +1148,7 @@ private void _______repeated_phrases___________________________________(){}
     
     public static final String USER_NAME_FOR_SELF = "Myself";
     public static final String USER_TEAM_FOR_INSTRUCTOR = "Instructors";
-    public static final String USER_NOT_IN_A_SECTION = "Not in a section";
+    public static final String USER_NOT_IN_A_SECTION = "No specified recipient";
```

Comment:
Assuming the above did not occur, the variable name should then be changed to `USER_NAME_FOR_NO_SPECIFIED_RECIPIENT` or alike.